src/Conversion/ONNXToStablehlo/ConvertONNXToStablehlo.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//====------ ConvertONNXToStablehlo.cpp - ONNX dialects to Stablehlo lowering
```

Comment:
NIT: could be reformatted to fit into one line, copyright date should probably be updated to 2024.